src/components/atoms/SpecialServiceCard/SpecialServiceCard.module.scss

```diff
@@ -0,0 +1,24 @@
+@use "sass:map";
+@import "../../../foundations/sass/all";
+
+.service-card {
+  @include center;
+  flex-direction: column;
+  
+  gap: 15px;
+  font-family: $playfair;
+
+  &__Title {
+    color: $Special-service-heading-color;
```

Comment:
You need to declare special-service-heading-color in the foundations/_color.scss file.